gremlin-test/src/main/java/org/apache/tinkerpop/gremlin/structure/FeatureSupportTest.java

```diff
@@ -587,7 +587,7 @@ public void shouldSupportRemoveEdgesIfEdgeCanBeRemoved() throws Exception {
         public void shouldSupportRemovePropertyIfAPropertyCanBeRemoved() throws Exception {
             try {
                 final Vertex v = graph.addVertex();
-                final Edge e = v.addEdge("self", v);
+                final Edge e = v.addEdge("self", v, "name", "foo");
```

Comment:
In all your thinking on this PR, did you ever think of any way to "test our tests" for proper feature assignments? perhaps that's a bit advanced and impossible a thing to do, but anything else we could do to make this process of adding features less error prone? or do we just need to rely on good code reviews of tests to be sure features go in the right way?